### PR TITLE
feat: add new fix for error in sending to subaddress

### PR DESCRIPTION
**Pull request by thanhdanh**
# Summary
Apply new fix of error when sending to subaddress at mymonero-core-cpp's PRs ([#30](https://github.com/ExodusMovement/mymonero-core-cpp/pull/30) and [#31](https://github.com/ExodusMovement/mymonero-core-cpp/pull/31))

Mirror of [react-native-fast-crypto#61](https://github.com/ExodusMovement/react-native-fast-crypto/pull/61)

### CODE REVIEW

File: monero_utils/MyMoneroCoreCpp_WASM.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ var MyMoneroCoreCpp = (function() {
 function(MyMoneroCoreCpp) {
```
Comment (Sekhmet) [inline monero_utils/MyMoneroCoreCpp_WASM.js:5]: I'm not sure how this repo works, but should we incldue wasm changes in each commit? .wasm file shouldn't be commited at all I think.

Comment (thanhdanh) [reply monero_utils/MyMoneroCoreCpp_WASM.js:5]: That's interesting. I think your idea could be inspired by `react-native-fast-crypto` when all built files are only included in the released package and would be ignored in the git repository.
Hi @RyanZim , how's about your idea?

Comment (thanhdanh) [reply monero_utils/MyMoneroCoreCpp_WASM.js:5]: FYI @gutenye

Comment (RyanZim) [reply monero_utils/MyMoneroCoreCpp_WASM.js:5]: At this point, WASM changes in git are the normal procedure; if we want to change that, let's discuss in slack, not here in public.

**Comment by feri42** [comment]
@thanhdanh You should try getting an utACK before merging. If you are having issues with that please feel free to ping me.

**Comment by thanhdanh** [comment]
> @thanhdanh You should try getting an utACK before merging. If you are having issues with that please feel free to ping me.

Yes, sure. I would do that the next time.

**Pull request merged**